CONTRIBUTING.md

```diff
@@ -10,10 +10,10 @@ Contributing to an open source project doesn't have to be through code alone. Th
 We're all just regular folks, those that write code, add comments, oversee the projects and of course those that use it. The rule of thumb is: if in doubt, **be kind**.
 
 ### Opening issues
-Filing issues with the project is a great step in helping the project - you're helping us learn what's not working, what you might want to see in the project or what we can do better. Even questions like 'how do I do X?' help us know where documentation may be lacking. When opening an issue, please make sure to follow the issue templates. Doing that helps us gather the required initial information to be able to further assist.
+Filing issues with the project is a great step in helping the project — you're helping us learn what's not working, what you might want to see in the project or what we can do better. Even questions like 'how do I do X?' help us know where documentation may be lacking. When opening an issue, please make sure to follow the issue templates. Doing that helps us gather the required initial information to be able to further assist.
```

Comment:
You can't see this in the diff's monospace font, but [the em-dash is a different punctuation mark](https://www.thepunctuationguide.com/em-dash.html), which makes more sense here than a hyphen.

(I learned this recently!)